httpcore/_async/connection.py

```diff
@@ -33,7 +34,9 @@ def __init__(
         self._http2_enabled = http2
         self._keepalive_expiry = keepalive_expiry
         self._uds = uds
-        self._ssl_context = SSLContext() if ssl_context is None else ssl_context
+        self._ssl_context = (
+            ssl.create_default_context() if ssl_context is None else ssl_context
```

Comment:
create_default_context creates a secure context but SSLContext() creates an insecure one

Reply:
is it work moving https://github.com/encode/httpx/blob/b43af721cd7804214396b5511e4f68d144a9b6a6/httpx/_config.py#L44 into httpcore?

Reply:
I think we'd like to have a minimal-as-possible SSL context in `httpcore`, eg...

https://github.com/tomchristie/httpcore-the-directors-cut/blob/main/httpcore/_ssl.py


```python
import certifi
import ssl


def default_ssl_context() -> ssl.SSLContext:
    context = ssl.create_default_context()
    context.load_verify_locations(certifi.where())
    return context
```

Reply:
I don't think `certifi.where()` is needed anymore in py3.6+